RecoLocalTracker/Phase2TrackerRecHits/plugins/Phase2TrackerRecHits.cc

```diff
@@ -63,8 +63,8 @@ void Phase2TrackerRecHits::produce(edm::StreamID sid, edm::Event& event, const e
   eventSetup.get< TrackerDigiGeometryRecord >().get(geomHandle);
   const TrackerGeometry* tkGeom(&(*geomHandle));
 
-  edm::ESHandle< TrackerTopology > tTopoHandle;
-  eventSetup.get< IdealGeometryRecord >().get(tTopoHandle);
+  //edm::ESHandle< TrackerTopology > tTopoHandle;
+  //eventSetup.get< IdealGeometryRecord >().get(tTopoHandle);
```

Comment:
if this commented out code is supposed to be reintroduced imminently, please leave a comment.
Otherwise, if it's not needed, please remove.